test/77.js

```diff
@@ -1,8 +1,19 @@
 const expect = require('chai').expect; 
 let solution = require('../solutions/77').solution;
+let solution1 = require('../solutions/77').solution1;
 
 describe('Return the count of vowels and consonants', () =>{
   it('should return { vowel: 2, consonant: 4 } for "yellow"', () =>{
     expect(solution('yellow')).to.eql({vowel: 2, consonant: 4 });
   });
+  it('should return { vowel: 3, consonant: 5 } for "maricris"', () =>{
+    expect(solution1('yellow')).to.eql({vowel: 2, consonant: 4 });
+  });
+  it('should return { vowel: 3, consonant: 4 } for "goodbye"', () =>{
+    expect(solution1('goodbye')).to.eql({vowel: 3, consonant: 4 });
+  });
+  it('should return { vowel: 2, consonant: 3 } for "bonzo"', () =>{
```

Comment:
Can we replace this test case with a different scenario - like a word without 0 vowel or 0 constant?